rpc/cloudagent/network/publicipaddress/moc_cloudagent_publicipaddress.proto

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache v2.0 license.
+
+syntax = "proto3";
+option go_package = "github.com/microsoft/moc/rpc/cloudagent/network";
+package moc.cloudagent.network;
+
+import "google/protobuf/wrappers.proto";
+import "moc_common_common.proto";
+import "moc_common_networkcommon.proto";
+
+message PublicIPAddressRequest {
+	repeated PublicIPAddress PublicIPAddresss = 1;
+	Operation OperationType = 2;
+}
+
+message PublicIPAddressResponse {
+	repeated PublicIPAddress PublicIPAddresss = 1;
+	google.protobuf.BoolValue Result = 2;
+	string Error = 3;
+}
+
```

Comment:
This file was WIP, so please ignore the formatting here. Below are the public IP properties listed from moc-sdk-for-go.